src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -68,6 +68,9 @@ class BookStep : public StepImp<TIn, TOut, BookStep<TIn, TOut>>
 
     boost::optional<Cache> cache_;
 
+    // Offers that were not used in the reverse pass because they becaue unfunded
```

Comment:
becaue -> became?  Useful comment.

Just for my own education I'd like to talk through the scenario(s) where a previous step (presumably in the same strand?) causes an offer to become unfunded.  I have a couple of thoughts about how this might happen, but I'd like to find out the real scoop.  Thanks.